doc/Programming-guide/Key-platform-features/Configuration-management/index.md

```diff
@@ -437,208 +514,477 @@ The base schema is obtained by transforming the configuration schema, as follows
+## Override algorithm
+
+To define and apply the resulting configuration update for the endpoint, the Operations service proceeds as follows:
+
+1. Evaluates the endpoint group membership according to the endpoint profile.
+2. Merges all configuration data sets assigned to the groups the endpoint belongs to, starting with the one that has the lowest weight
+(which is always group "all" with the 0 weight). <br/>
+In case of the conflicting field values, the field is set with the value from the group with the highest weight.
 
-**NOTE:**
-In case of the arrays, the **overrideStrategy** field in the configuration schema defines the way in which the arrays are merged.
-Record UUID fields never change from the values in the lowest weight group they were first encountered in.
-Both profile schema and configuration schema versions the endpoint supports are taken into account during the merge.
+>**NOTE:**
+> In case of the arrays, the ```overrideStrategy``` field in the configuration schema defines the way in which the arrays are merged. <br/>
+Record UUID fields never change from the values in the lowest weight group they were first encountered in. <br/>
+Both profile schema and configuration schema versions the endpoint supports are taken into account during the merge. <br/>
 
-## User-specific configuration management
+# User-specific configuration management
 
-The user-specific configuration management allows updating configuration data for the specific user under the application. The user-specific configuration management implements the same approach as in [Group-specific configuration]() management, based on the override schema and override algorithm.
+The user-specific configuration management allows updating configuration data for the specific user under the application. The user-specific
+configuration management implements the same approach as in [Group-specific configuration](#group-specific-configuration-management) management,
+based on the override schema and override algorithm.
 
-**NOTE:**
-Since each endpoint belongs to only one user at a time, the override algorithm does not support data set merges as in the group-specific configuration management.  
+>**NOTE:**
+> Since each endpoint belongs to only one user at a time, the override algorithm does not support data set merges as in the group-specific
+configuration management. <br/>
 The override algorithm applies user-specific overrides only after all group-specific overrides have been applied.
 
-## Endpoint data synchronization
+# Endpoint data synchronization
 
-After calculating the up-to-date configuration for the endpoint, the Operations server constructs a delta update based on the knowledge of the prior configuration of that endpoint. In certain cases, the delta update can contain a complete endpoint configuration.
+After calculating the up-to-date configuration for the endpoint, the Operations service constructs a delta update based on the knowledge of the
```

Comment:
This is not true. please rewrite this. No delta-updates supported at the moment.